components/_share.scss

```diff
@@ -0,0 +1,170 @@
+.sharing-info {
+  clear: both;
+  margin-top: 50px;
+
+  @include span-columns(8 of 12);
+  @include shift(2);
+
+  h2 {
+    margin-bottom: $base-font-size;
+  }
+}
+
+.sharing-box {
+  clear: both;
+  border: 2px solid #F0F0F1;
```

Comment:
Color literals like `#F0F0F1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F0F0F1` should be written as `#f0f0f1`